app/controllers/api/auth_controller.rb

```diff
@@ -87,6 +87,10 @@ def sign_in
       return
     end
 
+    if !params[:email] or !params[:password]
```

Comment:
Would the recommended syntax here be `param[:email].nil?`, or is the ! operator fashionable as well?

Reply:
Also isn't  `||` recommended over `or`?

Reply:
> Would the recommended syntax here be `param[:email].nil?`, or is the ! operator fashionable as well?

Yeah, we could use `.nil?` here too. Both `!` and `.nil?` evaluate to `true` when not present.

> Also isn't `||` recommended over `or`?

I guess that in this case, since we are checking that both `email` and `password` are present, the precedence would not matter. Although, `||` is recommended over `or`, we even enforce this style on our `code-refactor` branch: https://github.com/standardnotes/syncing-server/blob/51878012e3b3df16a79b402c4f29ae83103fc70b/.rubocop-https---shopify-github-io-ruby-style-guide-rubocop-yml#L42